lib/cdo/rake_utils.rb

```diff
@@ -313,8 +313,11 @@ def self.local_environment?
     (rack_env?(:development, :test) && !CDO.chef_managed) || rack_env?(:adhoc)
   end
 
-  def self.wait_for_url(url)
-    system_ "until $(curl --output /dev/null --silent --head --fail #{url}); do sleep 5; done"
+  # Wait for 3 minutes by default
+  def self.wait_for_url(url, tries: 36, sleep_secs: 5)
```

Comment:
Can we make these constants?  Or parameters that are passed in from Drone to each build, so we can tweak them without a code change?